src/main/java/com/_up/megastore/controllers/interfaces/ICategoryController.java

```diff
@@ -4,15 +4,16 @@
 import com._up.megastore.controllers.responses.CategoryResponse;
 import jakarta.validation.Valid;
 import org.springframework.http.HttpStatus;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
 
 @RequestMapping("/api/v1/categories")
 public interface ICategoryController {
 
     @PostMapping @ResponseStatus(HttpStatus.CREATED)
     CategoryResponse saveCategory(@RequestBody @Valid CreateCategoryRequest createCategoryRequest);
 
+    @DeleteMapping(value = "/{categoryId}") @ResponseStatus(HttpStatus.OK)
+    CategoryResponse deleteCategory(@PathVariable UUID categoryId);
```

Comment:
Acá no deberíamos devolver nada si queremos mantener un estándar de todas las eliminaciones
```suggestion
    void deleteCategory(@PathVariable UUID categoryId);
```